recipes/openmeeg/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "2.4.0.dev" %}
+
+package:
+  name: openmeeg
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmeeg/openmeeg/archive/2.4-rc2.tar.gz
+  sha256: 254eb82717ffec8b79618c281fe5c87734b2f8d804ef369d414d91b1b71cd163
+
+build:
+  number: 0
+  skip: true  # [win32]
+  msvc_compiler: 14.0
+  features:
+    - vc14  # [win]
+
+requirements:
+  build:
+    - cmake >=3.8
+    - toolchain
+    - python
+    - numpy
```

Comment:
You link against numpy, you may want to build against the oldest version see https://conda-forge.org/docs/meta.html#building-against-numpy